modules/ROOT/pages/upgrade-index.adoc

```diff
@@ -1,35 +1,282 @@
+** Cluster-level components, which include Docker, Kubernetes, and the Ops Center. These components run on virtual machines (VMs) or bare-metal servers that act as nodes on the cluster.
++
+Cluster-level component upgrades are delivered as an updated installer package via the *Runtime Fabric downloads* page in Anypoint Runtime Manager.
 
-* Node configuration changes are reflected in newer versions of the installer scripts, and are also delivered via the `rtfctl` command-line utility. These changes include adjusting operating system limits and performing jobs to clean up system resources.
+** Runtime Fabric components, which include the agent used to connect Runtime Fabric to Anypoint Platform.
++
+Runtime Fabric component upgrades are obtained using a downloads page link that is displayed in the *Upgrade Available* and *Appliance Version* fields, as applicable, in Runtime Manager.
 
-* The `rtfctl` command-line utility upgrades are provided as a new binary that replaces the existing binary. The `rtfctl` utility is used to manage Runtime Fabric locally.
+* The `rtfctl` command-line utility, which is used to manage Runtime Fabric and perform the upgrades to cluster-level and node configuration components.
++
+Upgrades for the `rtfctl` utility are delivered as a new binary via the *Runtime Fabric downloads* page in Runtime Manager.
 
-Upgrades to cluster-level and node configuration components are performed using the `rtfctl` command-line utility.
+* Node configuration, which includes scripts for provisioning AWS and Azure infrastructure, preparing each node, bootstrapping a new Runtime Fabric, adjusting operating system limits, and performing jobs to clean up system resources.
++
+Node configuration upgrades are delivered as an updated install script version using the *Runtime Fabric downloads* page in Runtime Manager.
++
+[NOTE]
+If you are upgrading the node configuration with updated install scripts, the latest version of the cluster-level components (installer package) is automatically downloaded and used.
 
-## Downloads page
+It is important to run with the most current versions of the cluster-level (installer package) and `rtfctl` components.
 
-The latest versions of each downloadable package are found in the *Downloads* page, located in Anypoint Runtime Manager in the *Runtime Fabrics* tab. A link to each package’s release notes is provided to show the changes.
+== Determine the Component Versions You Are Using
 
-The *Runtime Fabric downloads* page also provides an installer link.
+Run the following command:
+```
+/opt/anypoint/runtimefabric/rtfctl version
+```
+
+Output includes:
+
+* The `rtfctl` component version in the `rtfctl` field
+* The installer package component version in the `appliance` field
+* The Runtime Fabric component version in the `agent` field
 
-If an appliance upgrade is available, a code snippet for performing the upgrade is provided, as shown in the following example:
+For example:
```

Comment:
Per our style guide, either end the statement introducing the example with a colon or reword to a more grammatically complete fragment or sentence than "For example:"